server/api/v1/endpoints/buoys.py

```diff
@@ -3,8 +3,8 @@
 
 from server.api.dependencies import get_db
 from server.core.constants import RELATIVE_ROOT
-from server.crud import buoy
-from server.schemas import Buoy
+from server.crud.crud_buoy import buoy
+from server.schemas.buoy import Buoy
```

Comment:
Why did you make this import change?

Reply:
To continue following absolute path convention. It doesn't make sense to have some files have absolute paths and leave the remaining ones with relative paths. That's my reasoning behind it. It's all or nothing when it comes to standardization and convention.

Reply:
If the intent is to follow absolute pathing convention, then your change seems unnecessary to me; it was already an absolute path.

Relative imports in Python are denoted with period(s) like so:

`from ..schemas.buoy import Buoy`

Reply:
It's just to be more specific. Because what if we had buoy class in more than one file in the schemas or crud directories?

Reply:
Good point. Yes, if that happened that would most definitely cause problems.

Should we allow that to happen though? Like why would we ever write a buoy class in more than one file in either schemas or crud directories? Doesn't that defeat the purpose of separation of concerns by relegating buoy class to its respective buoy file in its respective directory folder?

I think it's a safe decision to not worry about specificity because we shouldn't be mixing around classes and functionality like that in the first place.

Being more specific as you suggested is definitely more robust. I just don't think we should be worrying about supporting "anti-patterns" until such a problem arises. What do you think?